tests/snmpsim/samsungprinter_clx.snmprec

```diff
@@ -1 +1 @@
-1.3.6.1.2.1.1.1.0|4|Samsung CLX
+1.3.6.1.2.1.1.1.0|4|Samsung CLX-3170 Series;V1.00.01.64 Sep-27-2010;Engine 1.77.81;NIC V4.01.20(CLX-3170) 02-05-2010;S/N AAAAAAA000AAAA000
```

Comment:
where is all this new data coming from? same with all other snmprec changes

Reply:
These are from the stats site.  It logs sysObjectID and sysDescr